docs/xml/metainfo-desktopapp.xml

```diff
@@ -10,7 +10,8 @@
 	<section id="spec-appdata-introduction">
 		<title>Introduction</title>
 		<para>
-			A desktop application is an application which has a graphical user interface and is commonly used with mouse and keyboard. It also ships a Freedesktop
+			A desktop application is a software package which includes a graphical user interface as its primary method of interaction and is designed to be used with mouse and keyboard,
+			irrespective of whether it also supports different invocation target sizes for alternative methods of input, like capacitive touch devices. Its package must include a Freedesktop
 			<filename>.desktop</filename> file to be visible in application menus.
```

Comment:
```suggestion
			A desktop application is interactive software that presents a graphical interface to the user.
			To appear in application listings, the packaged application should include a <filename>.desktop</filename> file.
```

Related: [AppStream 1.0.3](https://lists.freedesktop.org/archives/appstream/2024-April/000417.html) added support for `plasma-mobile` as a desktop style -- so the term `desktop` as used here does indeed cater to more devices than those with mouse+keyboard.